src/app/Library/CrudPanel/Traits/Fields.php

```diff
@@ -479,6 +479,15 @@ public function getStrippedSaveRequest($request)
     {
         $setting = $this->getOperationSetting('strippedRequest');
 
+        // if an invokable class was passed
+        // eg. \App\Http\Requests\BackpackStrippedRequest
+        if (class_exists($setting)) {
+            $setting = new $setting();
+
+            return $setting($request);
```

Comment:
```suggestion
            return is_callable($setting) ? $setting($request) : abort(500, get_class($setting) . ' is not invokable.';
```

Reply:
Aaah yes, good catch @pxpm ! Better to help the dev out with a clear message 👏👏👏 But you forgot to close the parantheses there:

```suggestion
            return is_callable($setting) ? $setting($request) : abort(500, get_class($setting) . ' is not invokable.');
```